exporters/otlp/otlpmetric/otlpmetricgrpc/exporter.go

```diff
@@ -43,7 +44,9 @@ func (e *Exporter) Aggregation(k metric.InstrumentKind) aggregation.Aggregation
 // This method returns an error if called after Shutdown.
 // This method returns an error if the method is canceled by the passed context.
 func (e *Exporter) Export(ctx context.Context, rm *metricdata.ResourceMetrics) error {
-	return e.wrapped.Export(ctx, rm)
+	err := e.wrapped.Export(ctx, rm)
+	global.Debug("OTLP/gRPC exporter export", "Data", rm)
```

Comment:
Do we want to log the entire data structure on every export?  That might be overkill, even in debug mode.

Reply:
I think it is fine for debug mode. It will help you understand the flow of data and what that data is. It will be helpful in tracking down missing data.

Reply:
I'm going to keep this for now. If we decide we want to change the verbosity level in the future for this type of data we can address that then.